src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PackageRestore/PackageRestoreSharedJoinableTaskCollection.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Composition;
+using Microsoft.VisualStudio.Threading;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.PackageRestore
+{
+    [Export(typeof(PackageRestoreSharedJoinableTaskCollection))]
+    [ProjectSystemContract(ProjectSystemContractScope.UnconfiguredProject, ProjectSystemContractProvider.Private, Cardinality = ImportCardinality.ExactlyOne)]
+    internal class PackageRestoreSharedJoinableTaskCollection : IJoinableTaskScope
+    {
+        private readonly JoinableTaskCollection _joinableTaskCollection;
+
+        private readonly JoinableTaskFactory _joinableTaskFactory;
```

Comment:
Nit: You don't need the fields, just assign properties the values in the constructor. C# will automatically create fields for you under the covers.

Reply:
Done. Fields removed in this [commit ](https://github.com/dotnet/project-system/pull/7199/commits/d5a2a1f9b385e6a96d6d1c5fcbde5b7d0f19ebc5)